MatrixSDK/MXRestClient.h

```diff
@@ -617,6 +617,19 @@ NS_REFINED_FOR_SWIFT;
                            success:(void (^)(void))success
                            failure:(void (^)(NSError *error))failure NS_REFINED_FOR_SWIFT;
 
+/**
+ Delete an account_data event for the client.
+
+ @param type The event type of the account_data to delete (@see kMXAccountDataType* strings)
+ Custom types should be namespaced to avoid clashes.
+ @param success A block object called when the operation succeeds.
+ @param failure A block object called when the operation fails.
+
+ @return a MXHTTPOperation instance.
+ */
+- (MXHTTPOperation*)deleteAccountDataWithType:(NSString*)type
+                                      success:(void (^)(void))success
+                                      failure:(void (^)(NSError *error))failure NS_REFINED_FOR_SWIFT;
```

Comment:
If setting the `NS_REFINED_FOR_SWIFT` macro, you should set the refined swift variant in `MXRestClient.swift`. Alternatively you could just remove the macro.

Reply:
Thanks for spotting it! 
I think I don't need it actually.
Deleted here -> https://github.com/matrix-org/matrix-ios-sdk/pull/1651/commits/9582483df75f4daa77cdc612b8dcb04e49560e71